requirements.txt

```diff
@@ -9,4 +9,7 @@ regex
 # For XLNet
 sentencepiece
 # For XLM
-sacremoses
\ No newline at end of file
+sacremoses
+# For ROUGE
+nltk
+py-rouge
```

Comment:
I think we just add an other `requirements.txt` to the example subfolder

Reply:
Ok, I agree it’s cleaner